x/conflict/keeper/grpc_query_provider_conflicts.go

```diff
@@ -28,12 +29,16 @@ func (k Keeper) ProviderConflicts(c context.Context, req *types.QueryProviderCon
 			reported = append(reported, conflict.Index)
 		}
 
+		if conflict.VoteState == types.StateReveal {
```

Comment:
thats not what i meant:
		for _, vote := range conflict.Votes {
			if vote.Address == req.Provider && vote.Result == types.Commit{
				revealed = append(revealed, conflict.Index)
			}
		}

Reply:
done. see https://github.com/lavanet/lava/commit/54089d8f845662df2d4962ceab0dad3ab3bf7546